<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BrowserClass</class>
 <widget class="QDialog" name="BrowserClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>899</width>
    <height>724</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Browser</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon.ico</normaloff>icon.ico</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="TW_qiehuang">
     <property name="styleSheet">
      <string notr="true">QTabWidget::pane { /* The tab widget frame */
    border-top: 2px solid #123456;
}
QTabBar::tab
{   
	background-color: rgb(175, 177, 179);
    height: 27px;
    min-width: 88px;
    border: none;
    margin-left: 5px;
    color: #999999;

}

QTabBar::tab:selected
{
    
	background-color: rgb(242, 242, 242);
    font-family: &quot;Microsoft YaHei&quot; ,微软雅黑,SimSun,宋体;
    font-weight: bold;
    border: 1px solid #bcd7f2;
    border-top-left-radius: 2px;
    border-top-right-radius: 2px;
    color: #666666;
}

QTabBar::tab:hover:!selected
{
    background-image: url(:/images/tabbar_hover.png);
    border: 1px solid #bcd7f2;
    border-top-left-radius: 2px;
    border-top-right-radius: 2px;
    color: #666666;
}
QPushButton {
    border: 0px;
}</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Tab 2</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="QPushButton" name="B_Return">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>20</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton:pressed {
  padding-left:1px;
    padding-top:2px;
}
QPushButton:hover{
background-color: rgb(136, 136, 136);

}
</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>Resources/arrow-left.png</normaloff>Resources/arrow-left.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="B_Forward">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>20</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton:pressed {
  padding-left:1px;
    padding-top:2px;
}
QPushButton:hover{
background-color: rgb(136, 136, 136);

}
</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>Resources/arrow-right.png</normaloff>Resources/arrow-right.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="B_Refresh">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>10</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton:hover{
background-color: rgb(136, 136, 136);

}

QPushButton:pressed {
  padding-left:1px;
    padding-top:2px;
}</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>Resources/refresh.png</normaloff>Resources/refresh.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLineEdit" name="LE_wangye">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>20</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="B_load">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>22</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
	background-color: rgb(37, 185, 79);
 color:white; 
border:0px;
}
QPushButton:hover {
 
  border-radius: 10px;
	background-color: rgb(31, 145, 50);
    
    
}
QPushButton:pressed {
  padding-left:1px;
    padding-top:2px;
}</string>
           </property>
           <property name="text">
            <string>Load</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QWebEngineView" name="webEngineView">
         <property name="url">
          <url>
           <string>about:blank</string>
          </url>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="p_JinDu">
         <property name="styleSheet">
          <string notr="true">QProgressBar::chunk {
    background-color: #05B8CC;
    width: 20px;
}
QProgressBar {
    border: 2px solid grey;
    border-radius: 5px;
    text-align: center;
}</string>
         </property>
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Tab 1</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header location="global">QtWebEngineWidgets/QWebEngineView</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Browser.qrc"/>
 </resources>
 <connections/>
</ui>
